### DISCUSSION TITLE
Where is errors logs in Symfony 5.3.9?

### DISCUSSION THREAD

**Discussion by karaka200** [symfony/symfony#43402] [Q&A] [answered]
I use Symfony 5.3.9 and worked on `dev` mode ( ENV = dev )

I see `/var/logs/` folder but that folder is empty.

Do I need to install a package for showing errors logs?

**Comment by BahmanMehrdad** [comment, marked as answer]
You need to install the logger package :

`composer require logger`

and for debugging install the debug package (in dev mode) :

`composer require debug --dev`

**Reply by karaka200** [reply]
Thank you
